### PR TITLE
Create to-zeeshan.json

### PR DESCRIPTION
**Pull request by m-zeeshan42**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/to-zeeshan.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "description": "zeeshan's email",
+  "repo": "https://zeeshan.is-a.dev",
+  "owner": {
+    "email": "zeeshanashraf424@gmail.com"
+  },
```
Comment (CuteDog5695) [inline domains/to-zeeshan.json:6]: you need to add username line (username line is your Github username)

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by m-zeeshan42 — COMMENTED** [review]
Username added

File: domains/to-zeeshan.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "description": "zeeshan's email",
+  "repo": "https://zeeshan.is-a.dev",
+  "owner": {
+    "username": "m-zeeshan42"
+    "email": "zeeshanashraf424@gmail.com"
```
Comment (CuteDog5695) [inline domains/to-zeeshan.json:6]:
```suggestion
    "username": "m-zeeshan42",
    "email": "zeeshanashraf424@gmail.com"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by m-zeeshan42 — COMMENTED** [review]
I am sorry for these small mistakes. Now i have double checked everything.

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**